extraConfigSriov.py

```diff
@@ -86,6 +86,27 @@ def run(self, _) -> None:
         time.sleep(60)
         print(client.oc("wait mcp sriov --for condition=updated --timeout=50m"))
 
+        # Workaround for PCI realloc issue.
+        need_pci_realloc = False
+        for e in self._cc["workers"]:
+            ip = client.get_ip(e['name'])
+            if ip is None:
+                sys.exit(-1)
+            rh = host.RemoteHost(ip)
+            rh.ssh_connect("core")
+            res = rh.run("systemctl list-units --state=failed --plain --no-legend | grep \"switchdev-configuration-\w*-nm.service\"").out.strip()
```

Comment:
`if pattern in rh.run(...).out:`

Reply:
Just out of curiosity, this can't do regex right? We need to use the re package if we want regex?

Reply:
yes `import re`